app/pages/external/OfficeOfDeparturePage.scala

```diff
@@ -26,3 +26,17 @@ case object OfficeOfDeparturePage extends ReadOnlyPage[CustomsOffice] {
 
   override def toString: String = "officeOfDeparture"
 }
+
+case object OfficeOfDepartureInCL112Page extends ReadOnlyPage[Boolean] {
```

Comment:
Q. Is the idea of the ReadOnlyPage to allow us to save values to the cache where there is no question? I.e. inferred? 

If so should they be 'pages' or 'settables'? and in a different package?